server/controllers/session-auth.js

```diff
@@ -0,0 +1,96 @@
+const {
+  fetchSpotifyToken,
+  fetchTwitchToken,
+  fetchTwitchChannel,
+} = require("../auth");
+
+const {
+  spotifyClientId,
+  spotifyClientSecret,
+  twitchClientSecret,
+  twitchClientId,
+} = require("../config/credentials.json");
+
+const SessionAuthController = {
+  postClientTwitchAccessCode(req, res) {
+    const {
+      body: { accessKey },
+    } = req;
+
+    fetchTwitchToken(twitchClientId, twitchClientSecret, accessKey)
+      .then((twitchResponse) => {
+        const {
+          access_token,
+          expires_in,
+          refresh_token,
+          token_type,
+        } = twitchResponse;
+
+        req.session.accessKeys.twitchToken = {
+          access_token,
+          expires_in,
+          refresh_token,
+          token_type,
+          expirationDate: new Date(new Date().getTime() + expires_in * 1000),
+        };
+
+        return fetchTwitchChannel(access_token, twitchClientId);
+      })
+      .then((channelResponse) => {
+        console.log({ channelResponse });
+        const { _id: channelId } = channelResponse;
+        req.session.accessKeys.twitchToken.channelId = channelId;
+        res.status(200).json({ success: true });
+      })
+      .catch((err) => {
+        console.log({ err });
+      });
+
+    // TODO: Invoke auth endpoint and generate proper token
```

Comment:
Does this still need to be implemented before merging? If not, can we get rid of this comment?

Reply:
yes, let me nuke the comment